example/react-native/src/components/screens/HomeScreen.js

```diff
@@ -0,0 +1,70 @@
+import React, { Component } from "react";
+import { View, Text, StatusBar } from "react-native";
+import { RNMetricsService } from "aerogear-react-native-sdk";
```

Comment:
Can we call it just `MetricsService`?

Reply:
Yes, at the moment this is only to differentiate between the typescript core and the react-native implementation